app/static/css/custom/footer.css

```diff
@@ -52,7 +52,7 @@ footer .copyright{
 
 #footer-menu select {background: #282C35; border-color:#AAAAAA; color: #AAAAAA  }
 
-#footer-menu-options{
+#footer-menu > div > div:first-child{
```

Comment:
Can you please explain the code? What you did seems to be doing the same thing as before.